src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+import {
+  isCurrentVenueNGRequestedSelector,
+  isCurrentVenueNGRequestingSelector,
+} from "utils/selectors";
+
+import { IFRAME_TEMPLATES } from "settings";
+
+import { useSelector } from "hooks/useSelector";
+import { useUserIsVenueOwner } from "hooks/useUserIsVenueOwner";
+import { useUser } from "hooks/useUser";
+
+import { LoadingPage } from "components/molecules/LoadingPage";
+import { AdminVideo } from "components/molecules/AdminVideo";
+import { BannerAdmin } from "components/organisms/BannerAdmin";
+
+import "./VenueAdminPage.scss";
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+
+export const VenueAdminPage: React.FC = () => {
+  useConnectCurrentVenueNG();
```

Comment:
redundant

Reply:
It's not, this is a new route now and needs the connect. otherwise the venue is undefined and not in the redux.